infra/aws/README.md

```diff
@@ -0,0 +1,83 @@
+Building a cluster on AWS
+=========================
+
+This directory containes three subdirectories:
+
+* packer - used to build the base image
+* terraform - used to spin up EC2 virtual machines
+* ansible - used to configure the virtual machines
+
+# Packer
+
+The packer directory contains a Hashicorp Packer template used
+to build the base AMI disk image for AWS. 
+
+If you are using the Protocol Labs Test Infra AWS account, the
+AMI image has already been built and published.
+
+See [packer.io](https://www.packer.io/) for information on how
+to use Packer.
+
+# Terraform
+
+The terraform directory contains a Hashicorp Terraform configuration
+that can be used to provision resources needed to run a cluster
+on AWS - the EC2 virtual machines, an autoscaling group for the workers,
+plus a VPC (Virtual Private Cloud) for the network.
+
+The default cluster sets up a "manager" machine, another machine to run
+redis, and 2 worker machines for running tests on.
+
+First, you'll need to install the [Terraform](https://www.terraform.io/).
+
+You'll also need to have your AWS credentials (Access Key ID and Secret
+Access Key) setup in a standard location.
+
+In the `terraform` directory, create a `terraform.tfvars` file that looks
```

Comment:
Can you create a file `terraform.tfvars` with the name `terraform.tfvars-example` that people can c&p and just change the keys. Also, make sure to add that `terraform.tfvars` to gitignore, or many people will fall into the trap of commit and push private keys.

Reply:
It's already in .gitignore. I'll create the terraform.tfvars-example.